connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -274,3 +275,14 @@ message StatFunction {
   }
 }
 
+// Rename columns on the input relation.
```

Comment:
i just notice that we have a public api `DataFrame.withColumnsRenamed`, maybe change the name of the message?

Reply:
Let me take a look on the `DataFrame.withColumnsRenamed`. I am thinking if that API can re-use this message.